src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalDayCost = 40;
+  const basicDiscont = 20;
+  const extraDiscont = 50;
+
+  switch (true) {
+    case days < 3:
+      return rentalDayCost * days;
```

Comment:
good but why you don't use a variable for this multiplication?

```suggestion
      return rentalDayCost * days; // const staticCost = rentalDayCost * days; (use your own naming for it)
```

Reply:
> good but why you don't use a variable for this multiplication?

didn't think of it(

Reply:
remember to use variables for any action or dials which you used 2 or more times)